enterprise/models.py

```diff
@@ -1142,9 +1145,14 @@ def user(self):
         """
         try:
             return User.objects.get(pk=self.user_id)
+            # return User.objects.select_related('profile').get(pk=self.user_id)
```

Comment:
Deciding against this?

Reply:
This is closer to what we actually want, but making it work in unittests is kinda tricky, since we don't have a `UserProfile` to import into edx-enterprise (whereas with `User`, it's just a core Django model).  There's probably some way to use django model factories to do it, though.